superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -1273,5 +1273,11 @@ export const controls = {
     hidden: true,
     description: 'The number of seconds before expiring the cache',
   },
+
+  generic_checkbox: {
```

Comment:
This is the name that will be use in the saved slice's json. Having something generic here will make it harder if/when trying to generate slice dynamically, and harder to read the json and understand what that maps to. Also references in the backend/frontend code won't be as explicit. Let's use a name that describe the actual feature this triggers.